### DISCUSSION TITLE
Share templates between sites in a multisite network

### DISCUSSION THREAD

**Discussion by splinksl** [elementor/elementor#25172] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When you work on a multisite with many sites and you want everyone to use the same single post template (for example), you have to upload the template to all the child sites, and if you have many sites it becomes very complicated to maintain

### Describe the solution you'd like.

In the template display conditions, add the option to perform on the current site or on all sites

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by kingfisher64** [comment]
This would be extremely useful to share say header/footer design templates so only one design is needed for all sites on the multisite. The amount of time and maintenance required at this end would be enormous if you would consider this.

**Comment by photoprismci** [comment]
`copy-webpack-plugin` is not a standard dependency of this project — it is not listed in `frontend/package.json` and not referenced in `frontend/webpack.config.js` on the current `develop` branch.

So if your build is failing on that module, it most likely means you (or a patch you applied) added a `require("copy-webpack-plugin")` to `webpack.config.js` without also adding it to the dependencies.

To fix it:

```bash
cd frontend
npm install --save-dev copy-webpack-plugin
```

Then commit both the updated `package.json` **and** `package-lock.json` before re-running `make docker-local`.

### Why running `npm install` locally did not help

The `.dockerignore` at the repo root excludes `node_modules/`:

```
/frontend/node_modules/
/node_modules/
```

So your locally-installed modules are never copied into the Docker build context. Inside Docker, `make all` runs `npm ci` (Makefile line 298), which installs **only** what is listed in `package-lock.json`. If `copy-webpack-plugin` is not in the lock file, it will not be installed, regardless of what you have locally.

### TL;DR

1. Add `copy-webpack-plugin` to `frontend/package.json` (`npm install --save-dev copy-webpack-plugin` from the `frontend/` directory).
2. Commit the resulting `package.json` and `package-lock.json` changes.
3. Re-run `make docker-local`.
